packages/generator/src/generator.js

```diff
@@ -273,6 +350,19 @@ export default class Generator {
     return true
   }
 
+  async addPayloadScript (route, html) {
```

Comment:
Side note: later we may need CSP handling

Reply:
cc @clarkdo on this